libcnb/src/layer/handling.rs

```diff
@@ -747,6 +788,7 @@ mod tests {
                 metadata: GenericMetadata::default(),
             },
             ExecDPrograms::Overwrite(HashMap::new()),
+            Sboms::Keep,
```

Comment:
There are no ~unit tests added here. I presume that's because we're favouring the testing via the test buildpack?

If so, in the future were we going to migrate some of the other IO-heavy "unit" tests in this file to test-buildpacks too?

Reply:
Yeah, these changes here are just to make them work again. Unsure about the permutations we should check as well. I agree that we should have some strategy when to test what with which tooling. "If you need to create temporary directories and/or files it should be an integration test via test buildpacks" seems sensible to me. Thoughts?

Reply:
Yeah that works for me.